setup.sh

```diff
@@ -60,4 +60,4 @@ fi
 
 pip install -e runner/
 pip install -e .
-pip install pocketsphinx  # Optional, for comparison
+#pip install pocketsphinx  # Optional, for comparison
```

Comment:
remove this line completely

Reply:
Yes, it is intentional. For my usecase, pocketsphinx is not necessary.
Decided to comment instead of removing so that you guys can decide what to do.

Reply:
This line has been un-commented in commit "Restores pocketsphinx."
This is to avoid affecting other parts of Mycroft that are not relevat to this PR.